### DISCUSSION TITLE
Nuxt v3 requires Node v14?

### DISCUSSION THREAD

**Discussion by edwh** [nuxt/nuxt#18320] [Questions] [answered]
I'm trying to use Nuxt Bridge on an existing project, which was using node v12.13.1.  nuxt dev gives a very terse FATAL: Not supported error.

Installing node v14.18.1 seems to get further.  Is v14 a requirement?  If so then it would be good to document this clearly on https://v3.nuxtjs.org/getting-started/bridge

**Comment by danielroe** [comment, marked as answer]
Yes, it is a requirement - see https://v3.nuxtjs.org/getting-started/introduction#prerequisites. It may be worth including a link in the bridge setup docs too.

**Reply by edwh** [reply]
Thank you.  Now, of course, it seems obvious.  But I had skipped straight to the Bridge section.

It also might be worth splitting out the mandatory requirements (node version) from the recommended dev environment.  I use JetBrains rather than Visual Studio, and I think that might have contributed to me not reading that section properly.
